workflow/executor/resource.go

```diff
@@ -372,9 +372,13 @@ func runKubectl(args ...string) ([]byte, error) {
 	}()
 	var buf bytes.Buffer
 	if err := kubectlcmd.NewKubectlCommand(kubectlcmd.KubectlOptions{
-		Arguments:   args,
-		ConfigFlags: genericclioptions.NewConfigFlags(true),
-		IOStreams:   genericclioptions.IOStreams{Out: &buf, ErrOut: os.Stderr},
+		Arguments: args,
+		// TODO(vadasambar): re-visit this once https://github.com/kubernetes/kubernetes/pull/120024 is merged
+		ConfigFlags: genericclioptions.NewConfigFlags(true).
+			WithDeprecatedPasswordFlag().
```

Comment:
This is not needed?

Reply:
Thank you for the comment. 

I have tried matching the upstream default config: https://github.com/kubernetes/kubernetes/pull/120024/files#diff-7e82ada63a2e6664b803ac3ccdb84759ac97338ce807df63be2f81e3c9e0ba9cR95 

Once https://github.com/kubernetes/kubernetes/pull/120024 is merged we are going to end up using it anyway. I don't have a strong opinion on this. Happy to remove it for now if you want. Let me know.